studio/src/editor-panel.js

```diff
@@ -186,14 +187,51 @@ export default class EditorPanel extends LitElement {
         );
     }
 
+    getFragmentPropsToUse() {
+        const props = {};
+        this.fragment?.fields.forEach((field) => {
+            if (field.name === 'cardTitle' && field.values?.length) {
+                props.cardTitle = field.values[0];
+            }
+            if (field.name === 'variant' && field.values?.length) {
+                props.variantCode = field.values[0];
+            }
+        });
+        VARIANTS.forEach((variant) => {
+            if (variant.value === props.variantCode) {
+                props.variantLabel = variant.label;
+                props.surface = variant.surface;
+            }
+        });
+        return props;
+    }
+
     async copyToUse() {
```

Comment:
can you add unit test for copyToUse function with some examples?

Reply:
Done